app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1169,7 +1172,7 @@ public void addWidget(Widget aWidget) {
     }
 
     @Override
-    public void updateWidget(final Widget aWidget) {
+    public void updateWidget(@NonNull final Widget aWidget) {
```

Comment:
I don't think we can guarantee that `aWidget `will never be null.

Reply:
Then we need to fix the base class.